tensorflow/core/ops/array_ops.cc

```diff
@@ -410,6 +410,27 @@ axis: Dimension along which to unpack.  Negative values wrap around, so the
 output: The list of tensors unpacked from `value`.
 )doc");
 
+REGISTER_OP("UnravelIndex")
+    .Input("indices: int32")
+    .Input("dims: int32")
+    .Output("output: int32")
```

Comment:
Should indices be int64?

Reply:
Thanks. I have updated the kernel. Now both int32 and int64 are supported for `UnravelIndex` through template.